templates/tab-books.php

```diff
@@ -8,22 +8,24 @@
  */
 
 ?>
-
+<p>Books, ebooks, audio books, music, and videos</p>
 <div class="panel"></div>
 <ul id="books-target">
-	<li><label><input type="radio" name="books-target" value="barton">Barton</label></li>
-	<li><label><input type="radio" name="books-target" value="worldcat" checked="checked">Worldcat</label></li>
+	<li><label><input type="radio" name="books-target" value="localbooks">at MIT</label></li>
+	<li>
+		<label><input type="radio" name="books-target" value="worldcat" checked="checked">libraries worldwide</label>
+	</li>
 </ul>
 <p>Also try:
-	<a href="/bartonplus">BartonPlus</a>,
-	<a href="/barton-reserves">Course reserves</a>,
-	<a href="/barton-theses">Thesis</a>, or
-	<a href="#search-more">view more search options</a>
+	<a href="/barton">Barton Classic</a>,
+	<a href="/worldcat">WorldCat</a>,
```

Comment:
Not a blocker since I can remove it later, but we dropped Worldcat here (since it's essentially an option above).